frontend/src/clients/ReportClient.ts

```diff
@@ -0,0 +1,30 @@
+import { HttpClient } from '@src/clients/Httpclient'
+import { AxiosError, HttpStatusCode } from 'axios'
+import { InternalServerException } from '@src/exceptions/InternalServerException'
+
+export class ReportClient extends HttpClient {
+  reportResponse = {
+    velocity: {
+      velocityForSP: '2',
```

Comment:
Don't use the magic number, try to refactor it.

Reply:
Should no use mock data in prod code